mep/accounts/tests/test_accounts_commands.py

```diff
@@ -529,6 +528,19 @@ def test_source_info(self):
         assert info['manifest'] == footnote.bibliography.manifest.uri
         assert info['image'] == str(footnote.image.image)
 
+    def test_get_object_data(self):
+        # get a subscription with no subcategory and both dates
+        event = Event.objects.filter(
+            subscription__isnull=False,
+            start_date__isnull=False, end_date__isnull=False,
+            subscription__category__isnull=True) \
+            .first()
+        data = self.cmd.get_object_data(event)
+        assert data['event_type'] == event.event_label
+        assert data['currency'] == 'FRF'
+        assert 'member' in data
+        assert 'subscription in data'
```

Comment:
??? I assume that this was supposed to be
```python
        assert 'subscription' in data
```

Reply:
😆 🤦‍♀️

Reply:
lol, this is a really good one. are strings truthy in Python? this bug might have gone unnoticed for a looooong time

Reply:
yeah, non-empty string is truthy

one little misplaced single quote... 😆